account_keys/50407f1203d1cc7eeaa3826838cf0aa77f5c5c656ae0d13d8c8ec6527681aee8.json

```diff
@@ -0,0 +1 @@
+{"account_id":"50407f1203d1cc7eeaa3826838cf0aa77f5c5c656ae0d13d8c8ec6527681aee8","public_key":"ed25519:6QGf6G8Ts5UFogwJsQtyGpnyL94Neb61VX5aoVkb7YjR","private_key":"ed25519:4nszy92J8f7bBnsps6stb2y6jUtE58uy6FAyAaDShqKXkRGoH3gJu8tL1r5UFs5U66w6b9TqsvAqNa6EzpkD7dEP"}
```

Comment:
Did this work for you with different account_ids? For my setup and in the README it is the `"account_id"` for all account_keys files should match the `relayer_account_id` in `config.toml`

Reply:
fixed